docker-compose.yml

```diff
@@ -20,6 +20,11 @@ services:
     build:
       dockerfile: Dockerfile
       context: ./backend/
+    read_only: true
+    tmpfs:
+      - /var/cache/nginx:uid=101,gid=101
+      - /var/run:uid=101,gid=101
+      - /app/tmp:uid=101,gid=101
```

Comment:
Is this bit meant to be in the telemetry-frontend section?

Reply:
Yes, it is.
The 2 `/var` paths are to make nginx happy despite the whole container being read-only.
The `/app` is to make the frontend happy despite the whole container being read-only.

Reply:
That makes sense to me!

Reply:
> That makes sense to me! I guess app/tmp is because we need to write the env-config file on start.

Exactly. And I made it locked, so we can write it once, and then never again. And the script doing this is read-only as well so it cannot be altered to change this behavior,

Reply:
Sorry, I realise now that I wasn't clear above!

This `tmpfs` section is currently **not** being aplpied to the frontend docker image, so I believ it needs moving up a few lines to be in the correct place :)

Reply:
Oh you are totally right. I see the issue now. I will fix that.

Reply:
This is the mix up between the 2 PRs FE and BE :)
I wondered why it would even work (it should not) wihtout the tmpfs, the answer is that `read-only` also belongs to the frontend part. I will move all of that up.